supabase/tests/stats.test.sql

```diff
@@ -1,4 +1,4 @@
-
+/*
 create function tests.test_catalog_stats()
 returns setof text as $$
 begin
```

Comment:
can this be fixed by simply removing the partition creation / checks of the test? AFAIK that's the only thing that's really changed -- we no longer partition this table.